ROADMAP.md

```diff
@@ -110,7 +101,36 @@ Any dates listed below and the specific issues that will ship in a given milesto
 
 ## Released
 
-### [v0.14.0 - Hardening, robustness, v1beta1 APIs in core](https://github.com/crossplane/crossplane/releases/tag/v0.14.0)
+### v1.0.0 Stable v1 APIs, package dep resolution, Prom metrics, and more
+
+* Stable v1 APIs
+
+* Hardening and cleanup for v1.0
+  * Prometheus metrics for all binaries [#314](https://github.com/crossplane/crossplane/issues/314)
+
+* Composition
+  * Claim update propagation to its underlying composite resource [#1649](https://github.com/crossplane/crossplane/issues/1649)
+
+* Package Manager
+  * Basic dependency resolution for packages [#1842](https://github.com/crossplane/crossplane/issues/1842)
+    * i.e. automatically install the providers a configuration needs.
+
+* Providers
+  * AWS Provider
+    * more API types [crossplane/provider-aws#149](https://github.com/crossplane/provider-aws/issues/149)
+  * Helm Provider
+    * v1beta1 APIs
+
+  * Code Generation of Providers (work-in-progress)
+    * AWS ACK Code Generation of the Crossplane provider-aws
+      * [initial auto generated resources](https://github.com/crossplane/provider-aws/issues/149#issuecomment-718208201)
+    * Azure Code Generation of the Crossplane provider-azure
+      * [initial auto generated resources](https://github.com/matthchr/k8s-infra/tree/crossplane-hacking)
+    * Clouds that don't have code gen pipelines
+      * Wrap stateless Terraform providers (work-in-progress) [#262](https://github.com/crossplane/crossplane/issues/262)
+      * [initial auto generated resources](https://github.com/kasey/provider-terraform-aws/tree/master/generated/resources)
```

Comment:
we have crossplane-contrib repos for the 3 subprojects you can start using:
https://github.com/crossplane-contrib/provider-terraform-aws
https://github.com/crossplane-contrib/terraform-runtime
https://github.com/crossplane-contrib/terraform-provider-gen

Reply:
cool, thx 👍